src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/MultiNodeTreePickerConverter.cs

```diff
@@ -100,7 +100,9 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
                 return multiNodeTreePicker;
             }
 
-            return base.ConvertFrom(context, culture, value);
+            // [ML] - return default list as its always nice to avoid null reference exceptions on lists =)
```

Comment:
I dunno about this. You're might be right but Microsoft guidelines indicate always using base converters as a fallback. https://msdn.microsoft.com/en-us/library/ayybcxe5.aspx